en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/extensions/request-body-passing.md

```diff
@@ -0,0 +1,34 @@
+You can enable request body passing in Choreo Connect using below configurations.
+After enabling the request body passing feature, enforcer will get the request data 
+submitted with the API call. Then, you can implement a custom filter to perform customized
+operations based on the input data provided with the API call.
+
+Follow the instructions below to enable request body passing feature:
+
+1. Add following configurations under `router` section in `config.toml` of Choreo Connect to enable request body passing feature.
```

Comment:
Shall we combine both of these sentences because there's only one step. Also we can avoid the repitition of the phrase "enable request body passing feature"